packages/eslint-plugin-office-addins/src/utils/load.ts

```diff
@@ -43,25 +43,47 @@ function composeObjectExpressionPropertyIntoString(
 
 export function getLoadArgument(
```

Comment:
The name of this method is a off now that you are looking at all arguments and each argument could end up returning multiple fields that are going to be loaded.  Should use a name that reflects this like 'parseLoadArguments' or 'getLoadPropeties or 'getPropertyList' or something like that.